misc/engineering-notes.txt

```diff
@@ -125,7 +125,7 @@ widths:
    The operands have types and widths. The width system gives names to
    these features. Operand widths often vary with the x86 effective
    operand size calculation; This is where those widths are specified.
-   A width is called and "oc2-code" and maps to a lower level type called
+   A width is called an "oc2-code" and maps to a lower level type called
```

Comment:
bikeshedding: `an` can be omitted here.

Reply:
Keeping both articles in sentences of the form "an X is called a Y" feels easier to read to my non-native speaker brain :)